src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -25,89 +28,84 @@
+      DocumentContext json = JsonPath.parse(entity);
+      return new FacebookUser(
+        json.read("$.email"),
+        json.read("$.password"),
+        json.read("$.id"));
+    } catch (IOException | URISyntaxException e) {
+      PageObjectLogging.log(URI_SYNTAX_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
+      throw new WebDriverException(ERROR_MESSAGE);
     }
 
-    public HashMap<String, String> deleteFacebookTestUser(String userId) {
-        try {
-            HttpResponse response = deleteTestUser(userId);
-            String entity = EntityUtils.toString(response.getEntity());
-            return new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
-        } catch (IOException e) {
-            PageObjectLogging.log(URI_SYNTAX_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
-            throw new WebDriverException(ERROR_MESSAGE);
-        } catch (URISyntaxException e) {
-            PageObjectLogging.log(URI_SYNTAX_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
-            throw new WebDriverException(ERROR_MESSAGE);
-        }
+  }
 
+  public HashMap<String, String> deleteFacebookTestUser(String userId) {
+    try {
+      HttpResponse response = deleteTestUser(userId);
+      String entity = EntityUtils.toString(response.getEntity());
+      return new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
+    } catch (IOException | URISyntaxException e) {
+      PageObjectLogging.log(URI_SYNTAX_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
+      throw new WebDriverException(ERROR_MESSAGE);
     }
 
-    private String getURLcreateUser(String appId) {
-        String host = "https://graph.facebook.com/v2.6/";
-        String path = appId+"/accounts/test-users";
-        return host + path;
-    }
+  }
 
-    private String getURLdeleteUser(String userId) {
-        String host = "https://graph.facebook.com/v2.7/";
-        String path = userId;
-        return host + path;
-    }
+  private String getURLCreateUser(String appId) {
+    return String.format("https://graph.facebook.com/v2.6/%s/accounts/test-users", appId);
+  }
 
-    private ArrayList<BasicNameValuePair> getParams() {
-        PARAMS = new ArrayList<>();
-        PARAMS.add(new BasicNameValuePair("access_token", wikia_production_app_access_token));
-        return PARAMS;
-    }
+  private String getURLDeleteUser(String userId) {
+    return String.format("https://graph.facebook.com/v2.7/%s", userId);
+  }
 
-    private HttpResponse createTestUser(String appId) throws IOException, URISyntaxException {
-            URL url = new URL(getURLcreateUser(appId));
-            CloseableHttpClient httpClient = HttpClientBuilder.create().disableAutomaticRetries().build();
-            HttpPost httpPost = getHttpPost(url);
+  private ArrayList<BasicNameValuePair> getParams() {
+    ArrayList<BasicNameValuePair> PARAMS = new ArrayList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)